qiskit/converters/circuit_to_dag.py

```diff
@@ -17,7 +17,7 @@
 
 
 def circuit_to_dag(circuit, copy_operations=True, *, qubit_order=None, clbit_order=None):
-    """Build a ``DAGCircuit`` object from a ``QuantumCircuit``.
+    """Build a :class:`~.DAGCircuit` object from a :class:`~.QuantumCircuit`.
```

Comment:
Just a note, the leading `~` is not needed here as this simply truncates the path of the object. Since here there's no path, as you just used `.<Object>`, there's nothing to truncate (but it's not harmful afaik so no need to change) 🙂

Reply:
done!

Reply:
done!